api/src/main/java/mcoffin/rogue/MecanumDrive.java

```diff
@@ -0,0 +1,3 @@
+public interface MecanumDrive {
+  public void vectorMove(double driveSpeed, double strafeSpeed, double turnSpeed);
```

Comment:
I think it would be a little more clear here to make the following renames:
* `driveSpeed` -> `forwardSpeed`
* `strafeSpeed` -> `horizontalSpeed`